cmake/Modules/FindOpenBLAS.cmake

```diff
@@ -0,0 +1,64 @@
+
+
+SET(Open_BLAS_INCLUDE_SEARCH_PATHS
+  /usr/include
+  /usr/include/openblas
+  /usr/include/openblas-base
+  /usr/local/include
+  /usr/local/include/openblas
+  /usr/local/include/openblas-base
+  /opt/OpenBLAS/include
```

Comment:
I'm on OS X (and installed openblas with homebrew).  I had to add

    /usr/local/opt/openblas/include